app/src/main/java/org/piramalswasthya/cho/ui/commons/personal_details/PersonalDetailsFragment.kt

```
@@ -465,249 +496,172 @@ class PersonalDetailsFragment : Fragment() {
+                    isNewAbha = apiPatient.patient.isNewAbha,
+                    healthIdDetails = apiPatient.patient.healthIdDetails,
+                    faceEmbedding = apiPatient.patient.faceEmbedding
+                )
+
+                var saveSuccess = false
+                try {
+                    withContext(Dispatchers.IO) {
+                        patientRepo.insertPatient(patientToSave)
+                    }
+                    saveSuccess = true
+                } catch (e: android.database.sqlite.SQLiteConstraintException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)